tests/contest/contest/src/tests/root_readonly_true/root_readonly_tests.rs

```diff
@@ -0,0 +1,36 @@
+use anyhow::{Context, Ok, Result};
+use oci_spec::runtime::{ProcessBuilder, RootBuilder, Spec, SpecBuilder};
+use test_framework::{test_result, Test, TestGroup, TestResult};
+
+use crate::utils::test_inside_container;
+
+fn create_spec(readonly: bool) -> Result<Spec> {
+    let spec = SpecBuilder::default()
+        .root(RootBuilder::default().readonly(readonly).build().unwrap())
+        .process(
+            ProcessBuilder::default()
+                .args(vec!["runtimetest".to_string(), "root_readonly".to_string()])
+                .build()
+                .expect("error in creating config"),
+        )
+        .build()
+        .context("failed to build spec")?;
+
+    Ok(spec)
+}
+
+fn root_readonly_test() -> TestResult {
+    let spec_true = test_result!(create_spec(true));
+    let spec_false = test_result!(create_spec(false));
+    test_inside_container(spec_true, &|_| Ok(()));
+    test_inside_container(spec_false, &|_| Ok(()))
```

Comment:
I think we should split this into two tests, maybe `root_readonly_true_test` and `root_readonly_false_test`. Current impl is correct, but I feel it is logically better to have two tests to check two behaviors rather than one test checking both.

Reply:
@YJDoc2 
I split into tow test. Is this correct?
https://github.com/youki-dev/youki/pull/2976/commits/81d6128c54e1e8cf9cf92b182c6298be25fa6010